src/portable/portable.c

```diff
@@ -1065,6 +1103,10 @@ static int attach_unit_file(
                 _cleanup_(unlink_and_freep) char *tmp = NULL;
                 _cleanup_close_ int fd = -1;
 
+                r = mac_selinux_create_file_prepare_label(path, m->selinux_label);
+                if (r < 0)
+                        log_debug_errno(r, "Failed to set SELinux security context '%s' for '%s', ignoring: %m", m->selinux_label, path);
```

Comment:
hmm `mac_selinux_create_file_prepare_label()` already logs on its own, no need to do so here again